islandora_rdm.module

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @file
+ * Module file.
+ */
+
+/**
+ * Implements hook_theme().
+ */
+function islandora_rdm_theme() {
+  return [
+    'node__islandora_rdm_dataset__search_result' => [
+      'template' => 'node--islandora-rdm-dataset--search-result',
+      'preprocess functions' => ['islandora_rdm_search_result_preprocess'],
+      'base hook' => 'node',
+    ],
+  ];
+}
+
+/**
+ * Preprocess function.
```

Comment:
Add a slightly more detailed comment, like "Preprocess function for search result display
Get the abstract description and put it in $variables."